digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/RoadAddressesService.scala

```diff
@@ -59,4 +65,25 @@ class RoadAddressesService(val eventbus: DigiroadEventBus, roadLinkServiceImplem
       }
     }
   }
+
+  def getRoadAddressPropertiesByLinkId(assetCoordinates: Point, linkId: Long, roadLink: RoadLinkLike, oldProperties: Seq[Property]): Seq[Property] = {
+    val mValue = GeometryUtils.calculateLinearReferenceFromPoint(assetCoordinates, roadLink.geometry)
+    roadAddressDAO.getByLinkId(linkId, mValue, mValue).flatMap { ra =>
+      val addrMValue = ra.addrAt(mValue).toString()
+
+      val newRoadAddressProperties =
```

Comment:
These properties are already in database for the construction of the form, but always comes empty because we want to calculate the road address info each time we select the Bus Stop to always have the updated information instead fixed values on database